docs/en/operation/Journal.md

```diff
@@ -300,6 +300,23 @@ $ ./bin/alluxio fsadmin journal quorum remove -domain <MASTER | JOB_MASTER> -add
 
 3- Verify that the removed member is no longer shown in the quorum info.
 
+##### Transferring leadership to a specific master
+To aid in debugging and to add flexibility, it is possible to manually change the leader of an embedded journal cluster.
+
+1- Check current quorum state:
```

Comment:
markdown doesn't respect `1-` as numbered categories. use `1.` instead and same for `2` below

Reply:
fixed